cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -1,79 +1,54 @@
+Then('{string} RadioButton is set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('div:nth-child(2) input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton is not set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is not set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton size on preview is set to {string}', (radioButtonName, size) => {
+Then('{string} RadioButton size on preview is set to {string}', (position, size) => {
   if (size === 'small') {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '16px')
+    radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.css', 'width', '16px')
       .and('have.css', 'height', '16px');
   } else {
-    cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.css', 'width', '24px')
+    radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.css', 'width', '24px')
       .and('have.css', 'height', '24px');
   }
 });
 
-Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (radioButtonName, marginLeft, marginRight) => {
-  cy.radioButton(radioButtonName, 'div div span[data-element="help"]').should('have.css', 'margin-left', marginLeft)
+Then('{string} field help is set to fieldHelpInline and has margin-left set to {string} and has margin-right {string}', (position, marginLeft, marginRight) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find(FIELD_HELP_PREVIEW).should('have.css', 'margin-left', marginLeft)
     .and('have.css', 'margin-right', marginRight);
 });
 
-Then('{string} field help is not set to fieldHelpInline and has margin-left set to {string}', (radioButtonName, marginLeft) => {
-  cy.radioButton(radioButtonName, 'div div span[data-element="help"]').should('have.css', 'margin-left', marginLeft);
-});
-
-Then('{string} RadioButton {string} inputWidth is set to {string}', (position, name, width) => {
-  radioButtonByPosition(positionOfElement(position)).parent()
-    .should('have.css', 'width', `${width}px`);
-});
-
-Then('{string} RadioButton label width is set to {string}', (radioButtonName, width) => {
-  cy.radioButton(radioButtonName, 'label').should('have.css', 'width', `${width}px`);
+Then('{string} field help is not set to fieldHelpInline and has margin-left set to {string}', (position, marginLeft) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find(FIELD_HELP_PREVIEW).should('have.css', 'margin-left', marginLeft);
```

Comment:
here also move `.find(FIELD_HELP_PREVIEW)` into the `index.js` file

Reply:
fixed